compute_endpoint/globus_compute_endpoint/engines/thread_pool.py

```diff
@@ -22,11 +23,17 @@
 
 
 class ThreadPoolEngine(GlobusComputeEngineBase):
-    def __init__(self, *args, label: str = "ThreadPoolEngine", **kwargs):
+    def __init__(
+        self,
+        *args,
+        working_dir: str | os.PathLike = "tasks_working_dir",
+        label: str = "ThreadPoolEngine",
+        **kwargs,
+    ):
         self.label = label
         self.executor = NativeExecutor(*args, **kwargs)
         self._status_report_thread = ReportingThread(target=self.report_status, args=[])
-        super().__init__(*args, **kwargs)
+        super().__init__(*args, working_dir=working_dir, **kwargs)
```

Comment:
Similar as with the other classes, I don't see a need to collect and retransmit `working_dir`; suggest to let the base class handle it by default via `**kwargs**`.